packages/core/EventManager.ts

```diff
@@ -1008,4 +1020,26 @@ export default class EventManager {
       await crm.deleteEvent(reference.uid);
     }
   }
+
+  private generateAppOptions(eventTypeAppMetadata?: z.infer<typeof EventTypeAppMetadataSchema>) {
+    const appOptions: AppOptions = {
+      crm: {
+        skipContactCreation: [],
+      },
+    };
+
+    if (eventTypeAppMetadata) {
+      for (const key in eventTypeAppMetadata) {
+        const app = eventTypeAppMetadata[key as keyof typeof eventTypeAppMetadata];
+
+        if (app.appCategories && app.appCategories.includes("crm")) {
```

Comment:
Do we actually need to check for the app category here? Can't we just assume that if the app has a `skipContactCreation` then we should push it anyways?

Reply:
There's a subobject at appOptions also - I suppose you'd need to check crm one way or the other.

Reply:
app !== appOptions

Reply:
appOptions it's already formed at the top of the function. We don't need to re-check it.